test/command/suite/select/function/math_abs/int16_max.test

```diff
@@ -0,0 +1,14 @@
+plugin_register functions/math
+
+table_create Values TABLE_NO_KEY
+column_create Values value COLUMN_SCALAR Int16
+
+load --table Values
+[
+{"value":32767}
+]
+
+select Values \
+  --filter true \
+  --output_columns '_id,_score' \
+  --scorer '_score = math_abs(value)'
```

Comment:
It's better that we use `--output_columns 'math_abs(value)`. Because assignment to a variable may change value by unexpected cast.